balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -85,10 +76,6 @@ class MetricsServlet extends JacksonJsonServlet
       contentType = response.contentType
       response.result
     }
-    finally
-    {
-      periodMeter.update(System.currentTimeMillis() - begin, TimeUnit.MILLISECONDS)
```

Comment:
What do you envision going in the timer block? I thought the idea was to simply update a timer every time a request is processed (e.g. period-related timer gets updated every time a period request completes). I don't understand how that translates into a single callable block. The callable blocks are (to my understanding) recording the duration that it takes to run a Callable. From the codahale Timer:
```java
    /**
     * Adds a recorded duration.
     *
     * @param duration the length of the duration
     * @param unit     the scale unit of {@code duration}
     */
    public void update(long duration, TimeUnit unit) {
        update(unit.toNanos(duration));
    }

    /**
     * Times and records the duration of event.
     *
     * @param event a {@link Callable} whose {@link Callable#call()} method implements a process
     *              whose duration should be timed
     * @param <T>   the type of the value returned by {@code event}
     * @return the value returned by {@code event}
     * @throws Exception if {@code event} throws an {@link Exception}
     */
    public <T> T time(Callable<T> event) throws Exception {
        final long startTime = clock.getTick();
        try {
            return event.call();
        } finally {
            update(clock.getTick() - startTime);
        }
    }
```
It looks like the timer block corresponds to the "time" function, when we're trying to use the "update" function.